frontend/packages/kubevirt-plugin/src/components/modals/snapshot-modal/SupportedSnapshotVolumesList.tsx

```diff
@@ -5,7 +5,7 @@ import { useTranslation } from 'react-i18next';
 const SupportedSnapshotVolumesList = ({ supportedVolumes }: SupportedSnapshotVolumesListProps) => {
   const { t } = useTranslation();
 
-  if (supportedVolumes) {
+  if (!supportedVolumes) {
```

Comment:
I dont really know the code but is this  change relevant to the PR?

Reply:
This change is actually relevant to another bug that was solved on 4.10 
https://bugzilla.redhat.com/show_bug.cgi?id=2034647, 
which the modal is not showing the volumes that will be included in the snapshot, I thought it would be nice to fix that as well for 4.9

Reply:
I believe it should have been at least in a different commit with appropriate commit message, but thats just a suggestion :)

Reply:
I agree, if we back-port another bug, we should document it in bugzilla.

Reply:
removed, cloned [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=2034647) for 4.9 and opened in a new PR https://github.com/openshift/console/pull/10984